### ISSUE TITLE
python traceback - ansible-later v0.3.11

### ISSUE THREAD

**Issue by emsecrist** [thegeeklab/ansible-later#37]
I am using ansible-later v0.3.11 with ansible ansible-2.10.0 and ansible-base-2.10.1 and seeing the following traceback:
```
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
   File "/usr/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "/home/vsts/.local/lib/python3.6/site-packages/ansiblelater/__main__.py", line 94, in _review_wrapper
    return candidate.review(settings, lines)
  File "/home/vsts/.local/lib/python3.6/site-packages/ansiblelater/command/candidates.py", line 120, in review
    result = standard.check(self, settings.config)
  File "/home/vsts/.local/lib/python3.6/site-packages/ansiblelater/rules/ansiblefiles.py", line 197, in check_command_has_changes
    and "when" not in task["__ansible_action_meta__"]
KeyError: '__ansible_action_meta__'
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/vsts/.local/bin/ansible-later", line 8, in <module>
    sys.exit(main())
  File "/home/vsts/.local/lib/python3.6/site-packages/ansiblelater/__main__.py", line 80, in main
    errors = (sum(p.map(_review_wrapper, tasks)))
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 266, in map
```
Please let me know if further information is required.

**Comment by xoxys** [comment]
Thanks for reporting. Could you provide an example YAML file that causes this issue?

**Comment by emsecrist** [comment]
Thanks for the response, Here is the YAML code that causes the issue.
```
---
- name: run setup command for karaf
  shell:
    cmd: 'echo -e "feature:install wrapper\nwrapper:install -n talend-remote-engine\n" | ./trun'
    executable: /bin/bash
    chdir: /opt/talend_remote_engine/bin
    creates: /opt/talend_remote_engine/bin/talend-remote-engine.service
  failed_when: False
  environment:
    - JAVA_HOME: "/usr/lib/jvm/jre-11"
  notify: handler_restart_talend
...
```

**Comment by xoxys** [comment]
This should be fixed by https://github.com/thegeeklab/ansible-later/commit/479f6b98554e568a8b97b243d2e5114fe9402b9c. After the CI passed I'll tag a new release.

**Comment by xoxys** [comment]
Issue fixed in `v0.3.12`. Please test, thanks again for reporting!

**Comment by emsecrist** [comment]
Thanks for the quick help! My build passes now in my pipeline, using ```v0.3.12```
